### PR TITLE
(bugfix): fix false error msg `zinit-annex-patch-dl` is unavailable

### PR DESCRIPTION
**Pull request by vladdoster**

### CODE REVIEW

File: zinit-install.zsh [hunk 1]
```diff
@@ -1483,46 +1483,37 @@ builtin source "${ZINIT[BIN_DIR]}/zinit-side.zsh" || {
     reply=()
     for bpick ( "${bpicks[@]}" ) {
         list=( $init_list )
-
```
Comment (pschmitt) [inline zinit-install.zsh:1486]: Please undo! This hurts readability.

File: zinit-install.zsh [hunk 2]
```diff
@@ -2004,13 +1990,12 @@ zicp() {
 
     local cmd=cp
     if [[ $1 = (-m|--mv) ]] { cmd=mv; shift; }
-
+
```
Comment (pschmitt) [inline zinit-install.zsh:1993]: Please revert this.

**Comment by AndrewDaws** [comment]
Formatting changes to unrelated code should not be performed when addressing a single problem.

**Comment by vladdoster** [comment]
> Formatting changes to unrelated code should not be performed when addressing a single problem.

I'm well aware, thanks!

**Pull request merged**